Mapper/BaseMapper.php

```diff
@@ -59,14 +58,14 @@ public function getAdmin()
     /**
      * @param string $key
      *
-     * @return \Sonata\AdminBundle\Mapper\BaseMapper
+     * @return static
```

Comment:
I think that we should use `@return $this` instead of `@return static`.

In proposed [PSR-5 for phpdoc](https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#keyword) there is:

> $this, the element to which this type applies is the same exact instance as the current class in the given context. As such this type is a stricter version of static as, in addition, the returned instance must not only be of the same class but also the same instance.
This type is often used as return value for methods implementing the Fluent Interface design pattern.